python/tests/res/job_queue/test_job_queue.py

```diff
@@ -79,9 +81,13 @@ def test_kill_jobs(self):
             assert job_queue.is_running()
             
             threads = start_all(job_queue)
-         
-            job_queue.kill_all_jobs()
-            assert not job_queue.is_running()
+
```

Comment:
Should we add 

```python
time.sleep(3)
wait_until(
    lambda: self.assertTrue(job_queue.is_running())
)
```

to prevent the above `true`  vs `True` bug.

Reply:
fixed